### PR TITLE
Card Display with Hard Code data

### PR DESCRIPTION
**Pull request by Lordnickim**

### CODE REVIEW

File: client/src/components/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+export default function Card () {
+  const [productInfo, setProductInfo] = useState([])
+
+  const getProductInfo = () => {
+    axios.get('/product/productid')
```
Comment (erikeh) [inline client/src/components/Card.jsx]: Concern with adding too much "temporary" axios calls on client side. perhaps our priority should be to create the serverside axios calls first?

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import ReviewList from './ReviewList.jsx'
+import ReviewList from './ReviewList.jsx';
+import RelatedContainer from './RelatedContainer.jsx';
 
 class App extends React.Component {
```
Comment (erikeh) [inline client/src/components/App.jsx:5]: This is a general comment - if we're using hooks, should we keep the App component as a functional component as well?

**Review by raphaelspies — APPROVED** [review]

File: client/src/components/Card.jsx [hunk 3]
```diff
@@ -1,18 +1,11 @@
...
-  const getProductInfo = () => {
-    axios.get('/product/productid')
-      .then((rawData) => {
-        setProductInfo([rawData.data])
-      })
-      .catch((err) => {
-        console.log('Error getting Product...', err)
-      })
-  }
+export default function Card (props) {
```
Comment (erikeh) [inline client/src/components/Card.jsx:4]: Just learned that you can export default named functions if you use the traditional function declaration with the 'function' keyword. Interesting!

**Review by erikeh — APPROVED** [review]

**Comment by erikeh** [comment]
I'm a little iffy on adding too much clientside axios code. I think this is okay right now, but for the next time, i think we should start creating axios calls on the serverside first

File: client/src/components/Card.jsx (hunk 1 above)
Comment (Lordnickim) [reply client/src/components/Card.jsx]: @erikeh  Yea most of the calls will be on the server side primarly two types the styles and product. Biggest thing thou is For a single card it needs data from both styles and product then it needs then for Each card so gonna be quite a bit forEach kindof of calls with async and awaits mayber....

File: client/src/components/App.jsx (hunk 2 above)
Comment (Lordnickim) [reply client/src/components/App.jsx:5]: Lols didnt think about that Maybe????

File: client/src/components/Card.jsx (hunk 1 above)
Comment (erikeh) [reply client/src/components/Card.jsx]: Recommend creating a reusable method that gets data from both styles and product via promise.all() and returns combined data in useful format. 

That way every time you need to get both pieces of data you can just do something like

getCardData() 
  .then()....

**Pull request merged**